doist-base.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": [
+        "config:base",
+        "group:linters",
+        "group:monorepos",
+        "group:recommended",
+        "schedule:weekdays"
+    ],
+    "github-actions": {
+        "enabled": false
+    },
+    "stabilityDays": 5
+}
```

Comment:
Please add newline to end of file (will create an issue to configure prettier for this repo 😂)

Reply:
Or outright add it? 😁